src/main/resources/config/en.yml

```diff
@@ -352,3 +317,38 @@ MinecraftDiscordAccountLinkedMessageDeleteSeconds: 0
 ServerWatchdogEnabled: true
 ServerWatchdogTimeout: 30
 ServerWatchdogMessageCount: 3
+
+# HTTP proxy used for connecting to the Discord API
+# Leave this alone if you don't understand what it does
+ProxyHost: "https://example.com"
```

Comment:
This default value is bricked, should not contain the procotol at least for the okhttp implementation

Reply:
Would have bricked for `WebsocketFactory` too, fixed